cypress/support/utils.ts

```diff
@@ -221,20 +195,14 @@ export const replaceInCode = (term, replaceWith) => {
   }
 };
 
-/**
- * @param {number} line
- */
-export const jumpToLine = (line) => {
+export const jumpToLine = (line: number | string) => {
```

Comment:
Is there a use case for `string` here? Feel like all line numbers should be `number`

Reply:
We let you give a line number and a character number as a string and we've got one test that does that (e.g. "6:10"). Not sure if it'd be better to give `character: number` as a seperate argument?

Reply:
I would suggest separate utils, breaking out something like `jumpToPosition({ line: number, ch: number })`